src/tab.group.ts

```diff
@@ -274,18 +271,30 @@ export default class TabGroup extends LitElement {
     }
   }
 
+  #onDefaultSlotChange() {
+    this.#setAriaAttributes();
+  }
+
   #onNavSlotChange() {
-    this.#setUpTabs();
-    this.#updateTabsPanelsAndTabBar();
```

Comment:
Separated `#updateTabsPanelsAndTabBar()` into `#updateSelectedTabAndPanel()` and `#updateSelectedTabIndicator()` so I can call only what I need to (`#updateSelectedTabIndicator()`) in ` #onTabListResize()`.

Calling `#updateTabsPanelsAndTabBar()` in `#onTabListResize()` here also caused the `changes tabs on keyboard interaction` test to fail because it changes the selected tab `tabIndex` in the next tick. Happy to explain in more detail if you like.

Additionally, and for whatever it's worth, Tab Group, Tab, and Tab Panel are queued up for a rework, which they sorely need. They're the last of our early components that we've yet to rewrite.